examples/window-app/silabs/src/WindowManager.cpp

```diff
@@ -632,10 +634,15 @@ void WindowManager::UpdateLED()
     }
 }
 
+#ifdef DISPLAY_ENABLED
+void WindowManager::DrawUI(GLIB_Context_t * glibContext)
```

Comment:
What is the use of **glibContext** param?

Reply:
the callback needs it